lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -20,113 +53,183 @@
+                </div>
+                <div *ngIf="editableCategories" class="adf-metadata-categories-header">
+                    <adf-categories-management
+                        [(categoryNameControlVisible)]="categoryControlVisible"
+                        [disableRemoval]="saving"
+                        [categories]="categories"
+                        [managementMode]="categoriesManagementMode"
+                        [classifiableChanged]="classifiableChanged"
+                        (categoriesChange)="storeCategoriesToAssign($event)">
+                    </adf-categories-management>
+                </div>
+            </mat-expansion-panel>
         </ng-container>
-        <ng-container *ngIf="expanded">
             <ng-container *ngIf="groupedProperties$ | async; else loading; let groupedProperties">
                 <div *ngFor="let group of groupedProperties; let first = first;"
                      class="adf-metadata-grouped-properties-container">
-                    <mat-expansion-panel *ngIf="showGroup(group) || editable"
-                                         [attr.data-automation-id]="'adf-metadata-group-' + group.title"
-                                         [expanded]="canExpandTheCard(group) || !displayDefaultProperties && first">
+                    <mat-expansion-panel
+                                    [attr.data-automation-id]="'adf-metadata-group-' + group.title"
+                                    [expanded]="canExpandTheCard(group) || !displayDefaultProperties && first || group.expanded"
+                                    (opened)="group.expanded = true"
+                                    (closed)="group.expanded = false"
+                                    hideToggle>
                         <mat-expansion-panel-header>
-                            <mat-panel-title>
-                                {{ group.title | translate }}
-                            </mat-panel-title>
+                            <div class="adf-metadata-properties-panel-content">
+                                <mat-icon>
+                                    {{ group.expanded ? 'expand_more' : 'chevron_right'}}
+                                </mat-icon>
+                                <mat-panel-title class="adf-metadata-properties-title">
+                                    {{ group.title | translate }}
+                                </mat-panel-title>
+                            </div>
+                            <button *ngIf="!group.editable && !readOnly && hasAllowableOperations()"
+                                    mat-icon-button
+                                    [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate"
+                                    [attr.aria-label]="'CORE.METADATA.ACCESSIBILITY.EDIT' | translate"
+                                    data-automation-id="meta-data-card-toggle-edit" class="adf-edit-icon-buttons"
```

Comment:
For that button you have each property one under other, can you move class also under data-automation-id to have all properties for this button in column?

Reply:
@Yasa-Nataliya this is not addressed too